data/mil2525d/utilities/style-utilities/merge-stylx-utilities/Readme.md

```diff
@@ -3,38 +3,53 @@
-* An Style Icon specification file: [Military-All-Icons.csv](https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/imagefile_name_category_tags/Military-All-Icons.csv) with the following required fields: `styleItemCategory, styleItemName, styleItemUniqueId, styleItemTags`
+        *  This file is already included with this repo
+*  `(2)` A SQLite Engine/Editor/Interpreter capable of executing SQL commands from a script
+    *  These steps have been tested using [sqlite3.exe](http://www.sqlite.org/download.html) but you may use the application/sqlite engine of your choice
+*  `(3)` A Style Icon specification file: [Military-All-Icons.csv](https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/imagefile_name_category_tags/Military-All-Icons.csv) with the following required fields: `styleItemCategory, styleItemName, styleItemUniqueId, styleItemTags`
 
 ## Instructions 
 
-* Copy required files mentioned in Requirements above:
-    * `mil2525d-points-only.dat` 
-    * `mil2525d-lines-areas-labels-base-template.stylx` 
-    * `Military-All-Icons.csv`
-    * to this folder (`military-features-pro-data\source\utilities\merge-stylx-utilities`)
-* Rename  `mil2525d-lines-areas-labels-base-template.stylx` to `mil2525d.stylx`
-* Edit the versions.csv file to update the version information that will be inserted into the completed stylx file
-* Launch the Sqlite editor/interpreter of your choice
-* Follow the SQL commands/steps in [SqliteMergeStylx.sql](./SqliteMergeStylx.sql)
-    * Optionally you may edit this file to update the setting/string `{Full Path To}` to the local path and run this as an SQL script
-* Exit the Sqlite Editor
-* Test and update the resulting `mil2525d.stylx` in this repository
+* Copy/edit/verify the prerequisite files are present
+    * Copy the required "Points/Icons-Only" version of the .stylx mentioned in Requirements above to this folder and ensure it is named: `mil2525d-points-only.stylx`
+        * This file should now exist locally: `...utilities\style-utilities\merge-stylx-utilities\mil2525d-points-only.stylx`
+    * Clone the [joint-military-symbology-xml](https://github.com/Esri/joint-military-symbology-xml) repository locally if you have not already done so.
+        * This file should now exist locally: `...joint-military-symbology-xml\samples\imagefile_name_category_tags\Military-All-Icons.csv`
+    * Edit the versions.csv file to update the pertinent version information, this information will be inserted into the completed/merged mil2525d.stylx file
+        * This file should now be current with the updated/desired version information: `...utilities\style-utilities\merge-stylx-utilities\versions.csv`
+* Edit the automated scripts to point to the paths/configuration of your local machine:
+    * Edit the batch file: [AutomatedCopyandMerge.bat](./AutomatedCopyandMerge.bat), set/replace the following settings:
```

Comment:
It wasn't initially apparent what program I would be using to edit the .bat file, or the .sql file for that matter. However, once I opened it up in a text editor, the instructions on what to actually fix were very good.